core/src/test/kotlin/org/stellar/anchor/sep10/Sep10ServiceTest.kt

```diff
@@ -562,6 +562,33 @@ internal class Sep10ServiceTest {
     assertThrows<SepException> { sep10Service.createChallenge(cr) }
   }
 
+  @Test
+  fun testRequireKnownObnibusAccount() {
+    every { sep10Config.isRequireKnownOmnibusAccount } returns true
+    every { sep10Config.omnibusAccountList } returns listOf(TEST_ACCOUNT)
+    val cr = ChallengeRequest.of(TEST_ACCOUNT, TEST_MEMO, TEST_HOME_DOMAIN, null)
+
+    assertDoesNotThrow{ sep10Service.createChallenge(cr) }
```

Comment:
💡  The `Mockk` library has the `verify` function that can be used to verify that your mocks were called. We use that in some parts of our project, like in:
https://github.com/stellar/java-stellar-anchor-sdk/blob/ff95819513c0d381dc3322709ab5e4abcc316790/core/src/test/kotlin/org/stellar/anchor/sep31/Sep31ServiceTest.kt#L568-L571

This helps, among other things, to make sure you're not mocking anything unnecessarily, or that your mocked functions are being called with the correct parameters.